covid19-notebooks/nb-etl-run.sh

```diff
@@ -18,16 +18,6 @@ fi
 # echo "Running notebook $FILE..."
 # jupyter nbconvert --to notebook --inplace --execute "$FILE"
 
-echo "Running top10 script..."
```

Comment:
actually we can remove this file `covid19-notebooks/nb-etl-run.sh` as well as `covid19-notebooks/Dockerfile`, `covid19-notebooks/nb-etl-requirements.txt`, `covid19-notebooks/run-with-slack.sh` and the `bayes-by-county` folder, they're all for a deprecated job

Reply:
we can do it in a separate PR :)

Reply:
Will it be a part of this ticket? Or do we make another one?